src/utils/nviHelpers.ts

```diff
@@ -8,3 +17,125 @@ export const getNviYearFilterValues = () => {
   }
   return nviYearFilterValues;
 };
+
+const areTopLevelOrgsEqual = <T>(set1: Set<T>, set2: Set<T>) => {
```

Comment:
denne brukes ikke som generic nå
```suggestion
const areTopLevelOrgsEqual = (set1: Set<string>, set2: Set<string>) => {
```